src/Microsoft.AspNetCore.Mvc.Core/BindPropertyAttribute.cs

```diff
@@ -31,22 +31,14 @@ public virtual BindingSource BindingSource
 
                 return _bindingSource;
             }
-            protected set
-            {
-                _bindingSource = value;
-            }
+            protected set => _bindingSource = value;
```

Comment:
I don't mind it but it's weird to see `get` and `set` use different code styling.

Reply:
The VS rules only complains if it's a single line statement. I can change it back if the difference is jarring.